packages/element-lib/src/adapters/database/ElementCouchDBAdapter.js

```diff
@@ -0,0 +1,79 @@
+const NodeCouchDb = require('node-couchdb');
```

Comment:
This Adapter is almost certainly better for backend implementations, but we should note that it is possible to get a couchdb adapter that supports remote queries in the browser: https://github.com/redgeoff/slouch